gradle-plugin/src/main/kotlin/co/touchlab/kotlin/gradle/targets/native/tasks/CocoapodsTasks.kt

```diff
@@ -103,7 +103,13 @@ open class PodspecTask : DefaultTask() {
             |                    -P${KotlinCocoapodsPlugin.HEADER_PATHS_PROPERTY}="${'$'}HEADER_SEARCH_PATHS" \
             |                    -P${KotlinCocoapodsPlugin.FRAMEWORK_PATHS_PROPERTY}="${'$'}FRAMEWORK_SEARCH_PATHS"
             |            SCRIPT
-            |        }
+            |        },
+			|        {
+			|        	:name => 'Touch $specName.framework',
+			|        	:execution_position => :after_compile,
+			|        	:shell_path =>  '/bin/sh',
+			|        	:script => 'find "${'$'}{SRCROOT}" -type f -name *frameworks.sh -exec bash -c "touch \"{}\"" \;'
```

Comment:
I'm pretty sure we'll get some issue reports on this, but let's see how it goes.